model/model.py

```diff
@@ -0,0 +1,73 @@
+import pickle
+import json
+
+import pandas as pd
+from sklearn.ensemble import RandomForestRegressor  # type: ignore [import-untyped]
```

Comment:
Is this ignore comment needed?

Reply:
I get this mypy error otherwise: 
`model\model.py:5: error: Skipping analyzing "sklearn.ensemble": module is installed, but missing library stubs or py.typed marker  [import-untyped]`